cdc/pkg/cmd/cli/cli_changefeed_update.go

```diff
@@ -113,19 +112,6 @@ func (o *updateChangefeedOptions) run(cmd *cobra.Command) error {
 		cmd.Printf("%+v\n", change)
 	}
 
-	if !o.commonChangefeedOptions.noConfirm {
```

Comment:
The `noConfirm` will suppress the confirm prompt, will be useful for updating changefeed in scripts. Suggest to keep it.